README.md

```diff
@@ -138,8 +138,9 @@ Create stream class instance.
 - `format (str)` - source format with `None` (detect) as default. For the most cases format will be inferred from source.  See a list of supported formats below.
 - `encoding (str)` - source encoding with  `None` (detect) as default.
 - `skip_rows (int/str[])` - list of rows to skip by row number or row comment. Example: `skip_rows=[1, 2, '#', '//']` - rows 1, 2 and all rows started with `#` and `//` will be skipped.
-- `sample_size (int)` - rows count for table.sample. Set to "0" to prevent any parsing activities before actual table.iter call. In this case headers will not be extracted from the source.
 - `allow_html (bool)` - a flag to allow html
+- `sample_size (int)` - rows count for table.sample. Set to "0" to prevent any parsing activities before actual table.iter call. In this case headers will not be extracted from the source.
+- `bytes_sample_size (int)` - sample size in bytes for operations like encoding detection.
```

Comment:
It looks like `sample_size` (in rows) and `bytes_sample_size` (in bytes) are used to do similar things (perhaps that's not the case?). Can these be used together?

Reply:
Thanks. I've added a `bytes sample size` section to readme - https://github.com/frictionlessdata/tabulator-py/tree/bytes-sample-size#sample-size

Historically it's two different things. Not optimal though. On next major version I'd like to rework this sampling stuff.